src/Orleans.CodeGenerator/LibraryTypes.cs

```diff
@@ -227,10 +227,10 @@ INamedTypeSymbol Type(string metadataName)
         public INamedTypeSymbol Task { get; private set; }
         public INamedTypeSymbol Task_1 { get; private set; }
         public INamedTypeSymbol Type { get; private set; }
-        private readonly INamedTypeSymbol Uri;
-        private readonly INamedTypeSymbol? DateOnly;
-        private readonly INamedTypeSymbol DateTimeOffset;
-        private readonly INamedTypeSymbol? TimeOnly;
+        private INamedTypeSymbol _uri;
```

Comment:
I don't think "fixing" these helped in any way. And why was `readonly` removed in the process?